CHANGE_LOG

```diff
@@ -1,7 +1,18 @@
 Version 0.3
 -----------
+This release adds:
 
-In development
+* Dynamic generation of `scipy.special.signatures` (with thanks to Brandon T.
```

Comment:
```suggestion
* Dynamic generation of ``scipy.special.signatures`` (with thanks to Brandon T.
```
double backticks else you just get italics.

Reply:
fixed